<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>289</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/**/
QWidget{
    background-color:#ffe0ad;
    font-family:Poiret One;
    font: bold;
}

QToolTip {
    opacity: 300;
	border: 2px solid #ffc05c;
    border-radius: 4px;
    padding: 2px;
    background-color: #fdf1d9;
}

QFrame #frame_card{
    border: 5px solid #ffc05c;
		border-radius: 10px;
		background-color:#fdf1d9;
}
QFrame #frame{
		background-color:transparent;
}
QFrame #frame_2{
		background-color:transparent;
}

QLabel#label_title{
		border: 1px solid #ffc05c;
		border-top-left-radius: 5px;
		border-top-right-radius: 5px;
		background-color:#ffc05c;
    }

QLabel#label_image{
		border: 1px solid #ffc05c;
		border-top-left-radius: 5px;
		border-top-right-radius: 5px;
		background-color:#fdf1d9;
    }
QPushButton{
        background-color: #ffc05c;
        border-style: solid;
        border-width: 2px;
        border-radius: 13px;
        border-color: #ffc05c;
        font: bold 18px;
        /*min-width: 10em;*/
        padding: 0px;
        color:#1a5d75;
    }
    
    QPushButton:hover{
        border-color: #fe6d73;
    }
    
    QPushButton:pressed{
        border-width: 5px;
        border-radius: 17px;
        font:  15px;
        border-color: #fe6d73;
    }

QPushButton:checked{
        border-width: 5px;
        border-radius: 17px;
        font:  15px;
        border-color: #36a9d3;
    }
/**/
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_card">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="0" colspan="2">
       <widget class="QFrame" name="frame_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>45</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="leftMargin">
          <number>35</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>35</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_title">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>Title</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2" colspan="2">
       <widget class="QLabel" name="label_image">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QPushButton" name="pB_list">
           <property name="minimumSize">
            <size>
             <width>42</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>42</width>
             <height>42</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Identifier les ingr√©dients</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>images/icon_list.png</normaloff>images/icon_list.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
